libs/shared/cms/src/lib/queries/relying-party/types.ts

```diff
@@ -31,6 +31,9 @@ export interface RelyingPartyResult {
   SignupSetPasswordPage?: Page;
   SignupConfirmCodePage?: Page;
   SignupConfirmedSyncPage?: Page;
+  SigninPage?: Page;
+  SigninTokenCodePage?: Page;
+  SigninUnblockCodePage?: Page;
   Shared?: Shared;
```

Comment:
fyi in case my current open strapi related PR merges before this one: I changed that `Shared` to `shared` since it's lowercase in strapi and the generated code.